deploy/helm/minikube-dev.yml

```diff
@@ -4,9 +4,19 @@ global:
     maxUnavailable: 1
   exposeNodePorts: true
   image:
-    repository: "localhost:5000/data61/"
+    repository: "localhost:5000/data61"
     tag: "latest"
 gateway:
   auth:
     facebookClientId: "173073926555600"
     googleClientId: "275237095477-f7ej2gsvbl2alb8bcqcn7r5jk0ur719p.apps.googleusercontent.com"
+combined-db:
+  image:
+    tag: "latest"
+  waleBackup:
+    method: WAL
+    gsPrefix: "gs://magda-postgres-backups/test"
+    executionTime: 03:00
+    googleApplicationCreds:
+      secretName: storage-account-credentials
```

Comment:
Will this prevent the minikube deployment from working if the `storage-account-credentials` secret doesn't exist?

Reply:
Oops - yeah I forgot to remove the google storage config from minikube, ideally it shouldn't be using that at all unless you opt-in to it.